--- a/e2e/vite.config.mts	2026-09-14 18:52:38.450143379 +0800
+++ b/e2e/vite.config.mts	2026-09-14 18:52:41.594057254 +0800
@@ -63,6 +63,7 @@
         path.resolve(repoRoot, 'e2e/app/components/rtl-floating.html'),
         path.resolve(repoRoot, 'e2e/app/components/keyboard-critical.html'),
         path.resolve(repoRoot, 'e2e/app/components/mobile.html'),
+        path.resolve(repoRoot, 'e2e/app/components/screen-reader.html'),
       ],
     },
   },
