--- e2e/vite.config.mts	2026-09-13 22:05:23.064683835 +0800
+++ /tmp/e2evc-edit.mts	2026-09-14 23:07:27.140469621 +0800
@@ -63,6 +63,7 @@
         path.resolve(repoRoot, 'e2e/app/components/rtl-floating.html'),
         path.resolve(repoRoot, 'e2e/app/components/keyboard-critical.html'),
         path.resolve(repoRoot, 'e2e/app/components/mobile.html'),
+        path.resolve(repoRoot, 'e2e/app/components/wcag22.html'),
       ],
     },
   },
